common/templates/record.html

```diff
@@ -20,7 +20,12 @@
                   </record-action-bar>
 
                 <span ng-show="recordTableModel[$index].displayType == 'markdown'" id="entity-{{$index}}-markdown">
-                    <span class="markdown-container" ng-bind-html="recordTableModel[$index].page.content | trustedHTML"></span>
+                    <span ng-if="recordTableModel[$index].page.content == '' || recordTableModel[$index].page.content == null ">
+                        <span class="markdown-container" ng-bind-html="strNone | trustedHTML"></span>                        
```

Comment:
Replace this whole line with the HTML you wrote as a string in the directive. There's no need to put the HTML in the JS link function and bind it as HTML when the content is static. We know we can "trust it as HTML" because it's static content that we generate.

Reply:
changed.